src/tickit/adapters/interpreters/wrappers/joining_interpreter.py

```diff
@@ -49,8 +49,10 @@ async def _combine_responses(
                 An asynchronous iterable containing a single reply message.
         """
         response_list = [response async for response in responses]
-        response = self.response_delimiter.join(response_list)
-        return wrap_as_async_iterable(response)
+        # type checking AnyStr.join doesn't work for some reason
+        # https://github.com/microsoft/pyright/issues/5556
```

Comment:
This should be fixed in the next version of pyright but will leave the ignore here until we can test it.

Reply:
Good stuff